doc/STYLE_GUIDE.md

```diff
@@ -58,6 +58,10 @@
 * References to constructor functions should use PascalCase
 * References to constructor instances should be camelCased
 * References to methods should be used with parentheses: `socket.end()` instead of `socket.end`
+* To draw special attention to a note, adhere to the following guidelines:
```

Comment:
As I said,

> I don't have any strong opinions about the notes style except that it should be only one, so the formatting may be discussed in this PR.

I am completely fine with changing the style to what you prefer, a Vim macro will chew it easily 😄